css/elementor-icons.css

```diff
@@ -1851,14 +1851,18 @@
   content: "\e9c6";
 }
 
+.eicon-off-canvas:before {
+  content: "\e9c7";
+}
```

Comment:
Why was it moved?

Reply:
Agree, please just add the new icon only :)

Reply:
it is automatically generated. I add changes manually to scss and index.html